src/Components/RegisterForm.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace ARKEcosystem\Fortify\Components;
+
+use ARKEcosystem\Fortify\Components\Concerns\ValidatesPassword;
+use ARKEcosystem\Fortify\Models;
+use Domain\Collaborator\Models\Invitation;
+use Livewire\Component;
+
+class RegisterForm extends Component
+{
+    use ValidatesPassword;
+    public string $name = '';
+    public string $username = '';
+    public string $email = '';
+    public string $password = '';
+    public string $password_confirmation = '';
+    public bool $terms = false;
```

Comment:
Those should be kept in a `$state` array like all other components do.

https://ark.dev/docs/program-incentives/guidelines/laravel-and-php#state-management